junction/emailer.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from os import path
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+
+
+def send_email(to, context, template_dir):
+    """Render given templates and send email to `to`.
+
+    :param to: User object to send email to..
+    :param context:
```

Comment:
Add a line saying `dict containing which needs to be passed to django template` ?